composer.json

```diff
@@ -18,12 +19,13 @@
         }
     ],
     "require": {
-        "php": "^7.0 || ^5.6",
-        "cakephp/cakephp": "^3.7",
-        "wyrihaximus/html-compress": "^2.0 || ^1.0"
+        "php": ">=5.5.9",
+        "cakephp/cakephp": "^3.7 || 4.x-dev as 4.0.0",
+        "cakephp/chronos": "^1.2 || 2.x-dev as 2.0.0",
+        "wyrihaximus/html-compress": "^1.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.7",
+        "phpunit/phpunit": "^5.7|^6.0|^8.0",
```

Comment:
Makes sense, but why are you skipping 7?

Reply:
If I'm not mistaken, CakePHP 3 is not compatible with phpunit 7. Instead, CakePHP 4 requires at least phpunit 8.

Reply:
Yeah you're right 3 targets 5 and 6 while 4 targets 8 :+1: